packages/runtime/src/editor-preview.main.ts

```diff
@@ -1355,10 +1355,12 @@ export const createRoot = (
     dataSignal.update((data) => {
       return {
         ...data,
-        'URL parameters': {
-          ...window.toddle.locationSignal.get().query,
-          ...window.toddle.locationSignal.get().params,
-        } as Record<string, string>,
+        'URL parameters': component?.route
```

Comment:
In theory you should also check for the old definition of a page (a path property perhaps?), but we haven't seen any of those pages for a while I believe, so this is fine
Perhaps you could use the utility function though: isPageComponent